### PR TITLE
test: Add snapshot tests for Events and Day component

### PR DESCRIPTION
**Pull request by limoozyn**
Add snapshot tests for Events and Day component
--

**Checklist**

- [X] [YouTrack ticket](https://tutorium.myjetbrains.com/youtrack/issue/EDU-7) created
- [X] ready to review

**Why?**

To avoid accidental change of components

### CODE REVIEW

**Review by roman-holovin — CHANGES_REQUESTED** [review]
Where are snapshot files?

**Review by roman-holovin — APPROVED** [review]

File: src/test/components/Day/Day.test.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
 import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
 
 import Day from '../../../components/Day/Day'
```
Comment (vittorius) [inline src/test/components/Day/Day.test.js]: Why not an absolute import? Our Webpack config should treat the `src/` as a root dir.

File: src/test/components/Day/Day.test.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
```
Comment (vittorius) [inline src/test/components/Day/Day.test.js]: `ListSubHeader}`: missing space before the bracket

File: src/test/containers/Events/Events.test.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
```
Comment (vittorius) [inline src/test/containers/Events/Events.test.js]: ListSubHeader}: missing space before the bracket

File: src/test/containers/Events/Events.test.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
...
 import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
 
 import Events from '../../../containers/Events/Events'
```
Comment (vittorius) [inline src/test/containers/Events/Events.test.js]: The same thing with relative imports.

**Review by vittorius — CHANGES_REQUESTED** [review]

**Review by vittorius — APPROVED** [review]

**Pull request merged**